backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/cfg/ElasticsearchIndexSettings.java

```diff
@@ -117,7 +117,7 @@ private Defaults() {
 		}
 
 		public static final IndexStatus SCHEMA_MANAGEMENT_MINIMAL_REQUIRED_STATUS = IndexStatus.GREEN;
-		public static final int SCHEMA_MANAGEMENT_MINIMAL_REQUIRED_STATUS_WAIT_TIMEOUT = 10_000;
+		public static final int SCHEMA_MANAGEMENT_MINIMAL_REQUIRED_STATUS_WAIT_TIMEOUT = 60_000;
```

Comment:
While I think some users will need this value, I don't think we should changedefault.

Imagine a user just starting with the library. By default we require status "green" for all indexes. Imagine the user has only one Elasticsearch node: their indexes cannot be replicated, so they will never reach status "green" and be stuck at "yellow".

Now, without any configuration, startup with block waiting for this "green" status that will never happen; then it will issue an error stating that the status "green" was expected but wasn't reached in 10 seconds.

If you raise the default to 60s, the application will block for **one whole minute** instead... I'd say that's more than enough for most people to consider something went wrong and to kill the application. They'll never see the error message and will be left wondering why the hell everything is stuck.

I think it's better to have a smaller timeout by default, so that inexperienced users will get some feedback on what's wrong exactly. More experienced users that know their cluster is slow can raise the timeout, and *then* all the work you've done in this ticket will be very useful (not to mention it's also useful for long-running search queries anyway).

Reply:
the problem with that is we're using the same `SCHEMA_MANAGEMENT_MINIMAL_REQUIRED_STATUS_WAIT_TIMEOUT` property for the request (socket) timeout of health check calls.
In the master I think we have 60 seconds for this very value, so I reused here this value.